nondex-gradle-plugin/src/main/groovy/edu/illinois/nondex/gradle/plugin/tasks/NonDexHelp.groovy

```diff
@@ -0,0 +1,19 @@
+package edu.illinois.nondex.gradle.plugin.tasks
+
+import org.codehaus.groovy.reflection.ReflectionUtils
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+class NonDexHelp extends DefaultTask {
+    static final String NAME = "nondexHelp";
+
+    void init() {
+    	setDescription("NonDexHelp Description")
+        setGroup("NonDex")
+
+    	doFirst {
+    		println "\nNonDex Gradle Plugin"
+        	println ReflectionUtils.getCallingClass(0).getResourceAsStream("/nondexHelp.txt").text
```

Comment:
Can I just change this to println "help message" instead of loading the message string from another file?